wai-aria/scripts/aria-utils.js

```diff
@@ -146,5 +146,36 @@ const AriaUtils = {
   },
 
 
+  /*
+  assert_not_equals() wrapper function to verify an element's name computation
+  ignores aria-label (computedLabel != aria-label value).
+
+  Ex: <button aria-label="" data-testname="button with empty aria-label does not use aria-label as name">
+        class="ex">
+        buttonText
+        </button>
+
+      AriaUtils.verifyAriaLabelIsIgnored(".ex")
+
+  See Web Platform Test's test harness API documentation for available assert functions:
+  https://web-platform-tests.org/writing-tests/testharness-api.html#assert-functions
+
+  */
+  verifyAriaLabelIsIgnored: function(selector) {
```

Comment:
This feels too specific a function to be necessary in a shared utils file. If it's needed for this test, it could be in the local `script` block.

Reply:
For example, maybe you want something more generalized like a method that verifies the value of a different data-* attr that will NOT be contained in the `computedlabel`?

Reply:
Agreed; created a new generic function `verifyAttributeIsIgnoredforComputedLabel` that takes an arbitrary `data-ignoredattributeforlabel` as a parameter: beeeb539ef03c1a346527083a93da1c668cc05d6.